doc/cephfs/multimds.rst

```diff
@@ -40,7 +40,7 @@ result of commands.
 
     # fsmap e5: 1/1/1 up {0=a=up:active}, 2 up:standby
 
-    ceph fs set max_mds 2
+    ceph fs set <cephf-name> max_mds 2
```

Comment:
[off topic] the ceph(8) needs to be updated with the newly added subcommands of "fs". like `set`, `flag set`, and `{add|rm}_data_pool`.